internal/run/stub.go

```diff
@@ -92,18 +94,22 @@ func (cs *CommandStubber) find(args []string) *commandStub {
 type CommandCallback func([]string)
 
 type commandStub struct {
-	pattern    *regexp.Regexp
-	matched    bool
-	exitStatus int
-	stdout     string
 	callbacks  []CommandCallback
+	exitStatus int
+	matched    bool
+	output     string
+	pattern    *regexp.Regexp
+	stdout     io.Writer
```

Comment:
This was necessary in order to not break tests that were already in place. As you pointed out above I used `cmd.Run()` instead of `cmd.Output()` in some situations which caused me to make this change. Going forward all new tests should be stubbed out with Go interfaces.